lib/utils/job-utils/command-parser.js

```diff
@@ -1028,6 +1028,18 @@ function commandParserFactory(Logger, Promise, _) {
             }
         }));
     };
+    
+    CommandParser.prototype.validateParser = function validateParser(cmd) {
+        var self = this;
+        if (self[cmd] !== undefined) {
+            return Promise.resolve(cmd);
+        }
+        var error = new Error("Command parser does not exist");
+        return Promise.reject({
+            command: cmd, 
+            error: error 
```

Comment:
Huh, guess it does work! The command info seems to be redundant anyways though, since it's right there in Error message?